deploy/package.py

```diff
@@ -5,10 +5,14 @@
 import re
 import argparse
 import shutil
-
+import json
+from pathlib import Path
 
 # Collect args
 parser = argparse.ArgumentParser(description='Arguments required to run packaging function')
+parser.add_argument('--modes',
+    type=str, required=False, default='batch-account',
+    help='Type of model run host either batch-account or aks')
```

Comment:
help comment for modes could be somthing like:

optional flags like batch-account, aks, no-posgres

Reply:
if modes='', what happens?
This PR will pass modes '' by default.

Reply:
Made modes to be default. Also, on the package.sh, I have added batch-account as default. Good catch!